update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -0,0 +1,237 @@
+/*
+This adds corresponding sia405 values for interlis import/export.
+*/
+
+-- Add columns
+
+ALTER TABLE qwat_vl.value_list_base ADD COLUMN sia405code TEXT;
+
+
+-- Populate columns
+
+-- TODO : Bedding material mapping complete [X] to check [ ]
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'aucun';
+UPDATE qwat_vl.bedding SET sia405code = 'Beton' WHERE value_fr = 'béton';
+UPDATE qwat_vl.bedding SET sia405code = 'Fliess' WHERE value_fr = 'géotextile';
+UPDATE qwat_vl.bedding SET sia405code = 'Betonkies' WHERE value_fr = 'gravier';
+UPDATE qwat_vl.bedding SET sia405code = 'Sand' WHERE value_fr = 'sable';
+UPDATE qwat_vl.bedding SET sia405code = 'unbekannt' WHERE value_fr = 'terre';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cistern SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.cover_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- Hydrant material : mapping complete [X] to check [ ]
+UPDATE qwat_vl.hydrant_material SET sia405code = 'Metall' WHERE short_fr LIKE 'F %';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.hydrant_material SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_inf SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_model_sup SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_output SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.hydrant_provider SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.leak_cause SET sia405code = 'unbekannt' WHERE ???;
```

Comment:
-- Matching to Schadenstelle.Art (and maybe also Ursache):
UPDATE qwat_vl.leak_cause SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
UPDATE qwat_vl.leak_cause SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
UPDATE qwat_vl.leak_cause SET sia405code = 'Bruch.Querbruch' WHERE value_fr = 'cassure franche';
-- UPDATE qwat_vl.leak_cause SET sia405code = 'Bruch.Scherbe' WHERE value_fr = 'xxx';
UPDATE qwat_vl.leak_cause SET sia405code = 'Bruch.Riss' WHERE value_fr = 'cassure longitudinale';
UPDATE qwat_vl.leak_cause SET sia405code = 'Loch' WHERE value_fr = 'pièce non étanche';
UPDATE qwat_vl.leak_cause SET sia405code = 'Mangel' WHERE value_fr = 'pièce non étanche';
-- not sure about the next one - could also be Bruch.Scherbe or Loch or weitere
UPDATE qwat_vl.leak_cause SET sia405code = 'Mangel' WHERE value_fr = 'conduite arrachée';
UPDATE qwat_vl.leak_cause SET sia405code = 'Mangel' WHERE value_fr = 'corrosion';
-- in addition Schadenstelle.Ursache has to be set to 'Korrosion'
UPDATE qwat_vl.leak_cause SET sia405code = 'Mangel' WHERE value_fr = 'raccordement non étanche';
-- in addition Schadenstelle.Ursache has to be set to 'Verbindungen'
UPDATE qwat_vl.leak_cause SET sia405code = 'Mangel' WHERE value_fr = 'pièce non étanche';
-- in addition Schadenstelle.Ursache has to be set to 'bauliche_Maengel'